task/02-numbers-tasks.js

```diff
@@ -159,8 +169,9 @@ function parseNumberFromString(value) {
  *   3,3,3   => 5.196152422706632
  *   1,2,3   => 3.741657386773941
  */
-function getParallelipidedDiagonal(a,b,c) {
-    throw new Error('Not implemented');
+function getParallelipidedDiagonal(a, b, c) {
+    //return Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2) + Math.pow(c, 2));
+    return Math.hypot(a,b,c);
```

Comment:
space between parameters

Please, try to follow these rules https://javascript.info/coding-style